test/scWETH.t.sol

```diff
@@ -745,6 +745,27 @@ contract scWETHTest is Test {
         assertEq(vault.balanceOf(treasury), perfFees, "perf fee must be the same");
     }
 
+    function test_invest_at_almostZeroLtv(uint256 amount) public {
+        amount = bound(amount, 1e18, 1e20);
```

Comment:
should we give 1e18 and 1e20 a name

could make it easier to understand at a glance